server/sonar-server/src/main/java/org/sonar/server/permission/ws/template/SetDefaultTemplateAction.java

```diff
@@ -90,7 +89,7 @@ private void doHandle(SetDefaultTemplateWsRequest request) {
       PermissionTemplateDto template = findTemplate(dbSession, request);
       checkGlobalAdmin(userSession, template.getOrganizationUuid());
       validateQualifier(qualifier, resourceTypes);
```

Comment:
Out of scope, but I wonder if using `resourceTypes` is needed since the removal of type extensibility (dev cockpit). Having a hardcoded constant for root qualifiers is good enough IMO.
cc @teryk